migrations/2_batch_exchange.js

```diff
@@ -1,7 +1,15 @@
-const migrateBatchExchange = require("../src/migration/PoC_dfusion")
+const migrateBatchExchange = require("../src/migration/migrate_BatchExchange")
+const { getArtifactFromBuildFolderOrImport } = require("../src/migration/utilities")
 
 module.exports = async function (deployer, network, accounts, web3) {
+  const BatchExchange = getArtifactFromBuildFolderOrImport(
```

Comment:
Here we can always use the artifact directly since this will only be called when we are in the native context.